puma/plot_base.py

```diff
@@ -216,7 +216,13 @@ def __post_init__(self):
             [None] * self.n_ratio_panels if self.ymax_ratio is None else self.ymax_ratio
         )
 
-        self.ylabel_ratio = ["Ratio"] * self.n_ratio_panels
+        if self.ylabel_ratio is None:
+            self.ylabel_ratio = ["Ratio"] * self.n_ratio_panels
+        if len(self.ylabel_ratio) != self.n_ratio_panels:
+            raise ValueError(
+                f"If you passed `ylabel_ratio` of length {len(self.ylabel_ratio)}, "
+                f"but n_ratio_panels of {self.n_ratio_panels}"
```

Comment:
The error message doesn't make sense

Reply:
Sorry for a delay in replying, was busy with other stuff.

Yeah I messed up the error message from too much re-writing.
My logic was that if one provides _N_ ratio panel labels, but doesn't have _N_ ratio panels, it should not work. But looking at the rest of the `puma` code, I think multiple ration panels are only used for ROC curves so I can remove this check altogether.  For now I will just fix the error message.